### PR TITLE
Adding advanced user bundle recipie

### PR DESCRIPTION
**Pull request by NenadStef**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+###> nenadstef/advanced-user-bundle-recipe ###
```
Comment (maxhelias) [inline .gitignore:1]: This file should not be there

Comment (Nyholm) [reply .gitignore:1]: Could you please remove this file?

File: nenadstef/advanced-user-bundle/1.0/config/packages/advanced_user.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+advanced_user:
```
Comment (Nyholm) [inline nenadstef/advanced-user-bundle/1.0/config/packages/advanced_user.yaml:1]: Could you rename this file to `nenadstef_advanced_user.yaml`?

**Comment by Nyholm** [comment]
Friendly ping

**Comment by Nyholm** [comment]
Im closing this due inactivity. Feel free to reopen this PR or to create a new one.

**Pull request closed without merging**